packages/bpk-component-floating-notification/src/BpkFloatingNotification.js

```diff
@@ -88,7 +88,6 @@ const BpkFloatingNotification = (props: Props) => {
       timeout={400}
       appear={animateOnEnter}
       exit={animateOnExit}
-      unmountOnExit
```

Comment:
We didn't appear to be using this correctly. Keeping this in prevents the appear animation animating.

On exit the component still removes from the dom:


https://github.com/Skyscanner/backpack/assets/28352715/9f184577-fadf-4b31-8279-f3ccd2351164